docs/glossary.md

```diff
@@ -270,6 +270,10 @@ A proposal to change the consensus rules of the Decred blockchain. Rule change p
 
 A 256-bit (32-byte) cryptographically secure random number that can be used to recover a wallet. When creating a wallet using any Decred wallet software, a 33 word "seed phrase" is also created from the 32-byte seed. If the wallet is lost or corrupted, the seed phrase can be used to restore the wallet's private keys, transaction history, and balances using any Decred wallet on any computer.
 
+#### Signature
+
+A cryptographic mechanism that proves that the owner of the private key has approved a transaction.
```

Comment:
Signatures are used for more than transactions.

Reply:
How about,

> A cryptographic mechanism that proves that the owner of the private key has approved a transaction or other data (e.g. the hash of a file).

Reply:
@jrick's version sounds good to me.

Reply:
@davecgh , @jrick , have implemented latest suggestion. Build passing, ready to merge.